metadata.json

```diff
@@ -12,26 +12,16 @@
     }
   ],
   "operatingsystem_support": [
-    {
-      "operatingsystem": "CentOS",
-      "operatingsystemrelease": [
-        "7",
-        "8",
-        "9"
```

Comment:
did you intentionally drop support for 8 and 9 as well? 9 is still supported.

Reply:
I thought we had agreed to simple delete centos?

Reply:
No, we still test on CentOS 9 stream, or CentOS Stream 9 or whatever it's called. We only dropped CentOS 7 and 8 because both are EoL: https://voxpupuli.org/blog/2024/05/08/CentOS-7-CentOS-Stream-8-deprecation/